setup/classes/class.ilSetupLanguage.php

```diff
@@ -41,7 +43,7 @@
  *  The format functions do not belong in class.Language. Those are also applicable elsewhere.
  *  Therefore, they would be better placed in class.Format
  */
-class ilLanguage
+class ilSetupLanguage extends ilLanguage
```

Comment:
@klees this is not related to the Artifact-thingy, isn't it? Absolutely fine for me, but there seem to be different declarations e.g. for `txt(...)`.

Reply:
PHP yelled at me, because both classes have been loaded the same time when I used that Artifact-thingy. So its kind of related, but also not exactly =)

I expect this to be an intermediate solution, and you are of course absolutely right regarding the declaration. This is some of the technical depth in the setup that needs to be fixed.